SampleCode/Python/sample_figure_understanding.ipynb

```diff
@@ -0,0 +1,563 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Figure understanding & hierarchical document structure analysis\n",
+    "\n",
+    "This notebook demonstrates an example of using Azure AI Document Intelligence to ouptut detected figures and hierarchical document structure (in markdown). It will then crop the figures and send image content (with its caption) to Azure Open AI GPTv model to understand the semantics. The image description will be used to update the markdown output, which can be further used for [semantic chunking](https://aka.ms/doc-gen-ai).\n",
```

Comment:
Just curious, are we calling GPT with Vision model as GPTv?

Reply:
Updated with GPT4-V